jetty-websocket/javax-websocket-common/src/main/java/org/eclipse/jetty/websocket/javax/common/JavaxWebSocketFrameHandler.java

```diff
@@ -217,7 +217,9 @@ public void onError(Throwable cause)
 
         if (errorHandle == null)
         {
-            LOG.warn("Unhandled Error: Endpoint " + endpointInstance.getClass().getName() + " missing onError handler", cause);
+            LOG.warn("Unhandled Error: Endpoint " + endpointInstance.getClass().getName() + " : " + cause);
+            if (LOG.isDebugEnabled())
```

Comment:
Remove DEBUG here.
If you don't want the stacktrace, add your own onError handler.
JSR356 is expected to be noisy if you don't follow the rules.